package.json

```diff
@@ -23,7 +23,8 @@
         "serve": "grunt serve",
         "setVersion": "node ./tools/release-tools/setVersion.js",
         "purge": "node common/scripts/install-run-rush.js purge",
-        "fullClean": "git clean -xdf && npm install && rush update --recheck --full",
+        "rushClean": "node common/scripts/install-run-rush.js clean",
+        "fullClean": "npm run rushClean && git clean -xdf && npm install && rush update --recheck --full",
```

Comment:
Splitting out the clean into a `rush clean` and then `git clean -xdf` as my machine (64gb) was running out of memory when performing a fully populated repo using `git clean -xdf`...